src/raftstore/store/peer.rs

```diff
@@ -50,6 +50,14 @@ use super::metrics::*;
 
 const TRANSFER_LEADER_ALLOW_LOG_LAG: u64 = 10;
 
+/// The return states of checking whether a peer is stale.
```

Comment:
IMO, it's better to write the comment as noun here, since the comment is for a enum definition.
For function definition, the comment usually starts with a verb.